apps/kyc-server/src/libs/websocket.ts

```
@@ -0,0 +1,189 @@
+interface KYCWebSocketData {
+	clientId: string
+	joinedAt: string
+	userId?: string
+}
+
+// Define the structure for KYC status record
+interface KYCStatusRecord {
+	user_id: string
+	status: string
+	verification_level: string
+	[key: string]: any
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)